tests/ci-helper.test.ts

```diff
@@ -61,6 +62,12 @@ class TestCIHelper extends CIHelper {
         this.ghGlue.updatePR = updatePR;
         this.updatePRCalls = updatePR.mock.calls;
 
+        const addPRLabels = jest.fn( async (_:string, labels: string[]):
```

Comment:
I guess there is no linting rule to require a space here `_:string`.  I will let you decide if you want to update it.

Reply:
Good point. Fixed.